src/components/accordion/daikin-accordion.ts

```diff
@@ -40,18 +41,61 @@ export class DaikinAccordion extends LitElement {
     ${unsafeCSS(tailwindStyles)}
 
     :host {
-      display: block;
-      width: 100%;
+      min-width: 160px;
+    }
+
+    ::slotted(daikin-accordion-item) {
+      --divider-top-display: block;
+      --divider-bottom-display: none;
     }
 
-    ::slotted(daikin-accordion-item:not(:last-child)) {
-      border-bottom: 1px solid #cecece;
+    ::slotted(daikin-accordion-item:last-child) {
+      --divider-bottom-display: block;
     }
   `;
 
+  @queryAssignedElements({ selector: "daikin-accordion-item" })
+  private readonly _items!: readonly DaikinAccordionItem[];
+
+  private _handleKeyDown(event: CustomEvent<{ key: 1 | -1 }>): void {
+    const moveOffset = event.detail.key;
+
+    const items = this._items;
+
+    // Get focused item if any
+    const activeElement = document.activeElement;
+
+    const focusedItemIndex = activeElement
+      ? items.findIndex((item) => item.contains(activeElement))
+      : -1;
+
+    const checkIsDisabledElementAndFocus = (
+      index: number,
+      moveOffset: number
+    ) => {
+      const nextFocusItemIndex =
+        moveOffset === 1
+          ? index === -1 || index === items.length - 1
+            ? 0
+            : index + moveOffset
+          : index <= 0
+            ? items.length - 1
+            : index + moveOffset;
+
+      if (items[nextFocusItemIndex].disabled) {
+        checkIsDisabledElementAndFocus(nextFocusItemIndex, moveOffset);
+      } else {
+        items[nextFocusItemIndex].focus();
+      }
+    };
+
+    checkIsDisabledElementAndFocus(focusedItemIndex, moveOffset);
+    return;
+  }
```

Comment:
- Use loop rather than recursion
- Prevent infinite loop
- Make the name and signature of the custom event more easy to understand

```suggestion
  private _handleMoveFocus(event: CustomEvent<{ direction: "up" | "down" }>): void {
    const moveOffset = event.detail.direction === "up" ? -1 : 1;
    const items = this._items;

    // Get focused item if any
    const activeElement = document.activeElement;
    const focusedItemIndex = activeElement
      ? items.findIndex((item) => item === activeElement)
      : -1;
      
    // If there is no accordion focused, do nothing.
    if (focusedItemIndex < 0) {
      return;
    }

    // Focus on the first accordion that is enabled.
    for (let index = focusedItemIndex + moveOffset, i = 0; i < items.length; index += moveOffset, i++) {
      index %= items.length;

      const item = items[index];
      if (item.disabled) {
        continue;
      }

      item.focus();
      break;
    }
  }
```